documentation/modules/con-considerations-for-data-storage.adoc

```diff
@@ -8,9 +8,19 @@
 
 An efficient data storage infrastructure is essential to the optimal performance of {ProductName}.
 
-{ProductName} requires block storage and is designed to work optimally with cloud-based block storage solutions, including Amazon Elastic Block Store (EBS). The use of file storage (for example, NFS) is not recommended.
+Block storage is a good choice.
+File storage (for example, NFS) is not recommended.
 
-Choose local storage (local persistent volumes) when possible. If local storage is not available, you can use a Storage Area Network (SAN) accessed by a protocol such as Fibre Channel or iSCSI.
+By block storage, we do not mean using Kubernetes raw block volumes.
```

Comment:
Can we add this as `NOTE:` or something? I think it looks weird when reading. I would maybe also phrase it as _NOTE: Strimzi does not require raw block volumes._. Because saying _By A we do not mean B_ is a bit strange when reading it.

Reply:
Rewritten. I've also moved it under our recommended choices.